css-three-column-layout/styles.css

```diff
@@ -0,0 +1,51 @@
+h2 {
+  color: white;
+  font-family: Lato, sans-serif;
+  font-weight: lighter;
+}
+
+.container {
+  background-color: #1297db;
```

Comment:
Instead of applying the `background-color` to the `container`, I would apply it to the `body` so that it fills the whole page